docs/sources/api/_index.md

```diff
@@ -939,7 +939,7 @@ The example belows show all possible statistics returned with their respective d
       },
       "summary": {
         "bytesProcessedPerSecond": 0, // Total of bytes processed per second
-        "execTime": 0, // Total execution time in seconds (float)
+        "execTime": 0, // Total execution time in nanoseconds (int)
```

Comment:
@ivanahuckova Do you think this will negatively affect Grafana's usage of our statistics?

Reply:
Kind of. When we are processing result's statistic, [we add a unit to the value](https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/result_transformer.ts#L321), so we can then display it in nice way. E.g: 

**if it is more than 60 sec, we will show 1 minute:**
![image](https://user-images.githubusercontent.com/30407135/148198387-0919abe7-0a39-4452-8c42-328b373187a9.png)

**if it is less than 1 sec, we will show ms**
![image](https://user-images.githubusercontent.com/30407135/148198227-be80e82e-e465-4157-a256-8a4a8ec22766.png)

So if we change this, we will need to add logic to check if the value is in `ns` (integer) or `sec`(float) which will be slightly hacky, but possible. 🙂

Reply:
Thank you @ivanahuckova 🙂 
As already discussed, both statistic time values `execTime` and `queueTime` will be returned by the HTTP API in second as float.